packages/forms-web-app/src/controllers/projects/documents/utils/filters/filter-params.js

```diff
@@ -0,0 +1,22 @@
+const removeIt = (key, value, removeKey, removeValue) => key === removeKey && value === removeValue;
+const isNotPaginationQuery = (key) => key !== 'page';
+const filterParams = (queryObj, removeMe) => {
+	const URL = new URLSearchParams();
+
+	for (const [key, value] of Object.entries(queryObj)) {
+		if (Array.isArray(value)) {
+			value.forEach((item) => {
+				if (!removeIt(key, item, removeMe.key, removeMe.value)) URL.append(key, item);
```

Comment:
struggled to flip this logic for some reason. 

Would be nice to say keepIt or something